testdata/baselines/reference/submodule/compiler/commonSourceDirectory.js.map.diff

```diff
@@ -0,0 +1,8 @@
+--- old.commonSourceDirectory.js.map
```

Comment:
I'm sort of wondering how useful this is when the maps are single lines so will always exist. Maybe we should do something special to format these at some point? (Might be annoying though).